public/content/de/basics/basic-types.md

```diff
@@ -0,0 +1,118 @@
+# Elementare Datentypen
+
+D bietet eine Vielzahl an elementaren Datentypen die immer die gleiche Größe haben -
+**unabhängig** von der Platform.
+Die einzige Ausnahme ist der Typ `real` für welchen der Kompiler
+die höchstmöglichste Genauigkeit für Gleitkommazahlen wählt.
+Es besteht kein Unterschied zwischen der Große einer Ganzzahl - unabhängig davon
+ob die Anwendung für ein 32-bit oder 64-bit Architektur kompiliert wird.
+
+<table class="table table-hover">
+<tr><td width="250px"><code class="prettyprint">bool</code></td> <td>8-bit</td></tr>
+<tr><td><code class="prettyprint">byte, ubyte, char</code></td> <td>8-bit</td></tr>
+<tr><td><code class="prettyprint">short, ushort, wchar</code></td> <td>16-bit</td></tr>
+<tr><td><code class="prettyprint">int, uint, dchar</code></td> <td>32-bit</td></tr>
+<tr><td><code class="prettyprint">long, ulong</code></td> <td>64-bit</td></tr>
+</table>
+
+#### Gleitkommatypen
+
+<table class="table table-hover">
+<tr><td width="250px"><code class="prettyprint">float</code></td> <td>32-bit</td></tr>
+<tr><td><code class="prettyprint">double</code></td> <td>64-bit</td></tr>
+<tr><td><code class="prettyprint">real</code></td> <td>abhängig von der Platform, 80-bit für Intel x86 32-bit</td></tr>
+</table>
+
+Der Präfix `u` kennzeichnet Typen ohne Vorzeichen (vom Englischen `unsigned`).
+Ein `char` ist ein UTF-8 Zeichen, `wchar` ein UTF-16 Zeichen and `dchar`
+ein UTF-32 Zeichen.
+
+Eine Umwandlung zwischen Variablen ist nur erlaubt wenn keine Präzision verloren
+gehen kann, obgleich die Umwandlung von Gleitkommatypen (z. B. `double` zu `float`)
+erlaubt ist.
+
+Eine Umwandlung zu einem anderen Typ kann durch den Ausdruck
+`cast(TYPE) <varName>` erzwungen werden. Dieser Ausdruck sollte aber mit großer
+Vorsicht verwenden werden, da mit `cast` Ausdrücke das Typsystem ausgeschaltet
+wird.
+
+Das spezielle Schlüsselwort `auto` erzeugt eine Variable und inferiert ihren Typ
+automatisch an Hand des Ausdruck of the rechten Seite. Zum Beispiel wird
+`auto i = 7` den Typ `int` deduzieren. Beachte, dass `auto` Typen trotzdem
+statisch während der Kompilerphase festgelegt werden und nicht den Typ ändern können
```

Comment:
`Kompilierphase` I think is better

Also shouldn't the rest of the sentence be more like `den Typen nicht nachträglich ändern können, wie für jede andere Variable mit explizitem Typen`